client/views/inventory/modals/editAssetModal/editAssetModal.coffee

```diff
@@ -86,6 +98,17 @@ checkUsername = (tpl, winCb, failCb) ->
         tpl.$('button[data-action=checkUsername]').html('<span class="glyphicon glyphicon-remove"></span>')
         if failCb then failCb()
 
+lookupShipDate = (tpl) ->
```

Comment:
Looks like this function is repeated in newAssetModal.coffee so it would be best to define it in one place and import it into these two locations.  Looks like some updates to the Meteor/package versions might be necessary to get that to work, so that can happen afterwards.